### PR TITLE
[DOCFIX]  Add instructions for Presto versions with support for SDS

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
alluxio-bot, check this please

**Comment by yuzhu** [comment]
alluxio-bot, check this please

**Comment by yuzhu** [comment]
@gpang PTAL

File: docs/en/core-services/Catalog.md [hunk 1]
```diff
@@ -213,7 +213,13 @@ and additions in the UDB tables.
 
 ## Using Alluxio Structured Data with Presto
 
-The Alluxio Catalog Service is built to be used with the connector in Presto.
+PrestoSQL version above 332 and PrestoDB version above 0.232 has built-in support for the Alluxio Catalog Service in their hive-hadoop2 connector.
```
Comment (gpang) [inline docs/en/core-services/Catalog.md:216]: I think this should be worded by `332 and above` or `333 and above` depending on which is true. This applies to `0.232` as well.

File: docs/en/core-services/Catalog.md [hunk 2]
```diff
@@ -249,6 +255,8 @@ Creating the `catalog_alluxio.properties` file means a new catalog named `catalo
 to Presto.
 Setting `connector.name=hive-alluxio` sets the connector type to the name of the
 new Alluxio connector for Presto, which is `hive-alluxio`.
+If you are using PrestoSQL version above 332 and PrestoDB version above 0.232, support for Alluxio Catalog Service is built into
```
Comment (gpang) [inline docs/en/core-services/Catalog.md:258]: Same clarification comment as above.

**Comment by yuzhu** [comment]
@gpang

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Comment by gpang** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**